docs/core/install/linux-ubuntu-2210.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Install .NET on Ubuntu 22.10
+description: Demonstrates the various ways to install .NET SDK and .NET Runtime on Ubuntu 22.10.
+author: adegeo
+ms.author: adegeo
+ms.date: 03/01/2023
+---
+
+# Install .NET SDK or .NET Runtime on Ubuntu 22.10
+
+This article discusses how to install .NET on Ubuntu 22.10; .NET 6 and .NET 7 are both supported.
+
+[!INCLUDE [linux-intro-sdk-vs-runtime](includes/linux-intro-sdk-vs-runtime.md)]
+
+[!INCLUDE [linux-install-package-manager-x64-vs-arm](includes/linux-install-package-manager-x64-vs-arm.md)]
+
+.NET is available in the Ubuntu package manager feeds, as well as the Microsoft repository. However, you should only one use or the other to install .NET. Review the [Decide how to install .NET](linux-ubuntu.md#decide-how-to-install-net) section to determine which feed you should use.
+
+> [!WARNING]
+> Don't use both repositories to manage .NET. If you've previously installed .NET from the Ubuntu feed or the Microsoft feed, you'll run into issues using the other feed. .NET is installed to different locations and is resolved differently for both package feeds. It's recommended that you uninstall previously installed versions of .NET and then install with the Microsoft package repository. For more information, see [How to register the Microsoft package repository](linux-ubuntu.md#register-the-microsoft-package-repository).
+
+## Supported versions
+
+The following versions of .NET are supported or available for Ubuntu 22.10:
+
+| Supported .NET versions | Available in Ubuntu feed | [Available in Microsoft feed](linux-ubuntu.md#register-the-microsoft-package-repository) |
+|-------------------------|--------------------------|-----------------------------------|
+| 7.0, 6.0                | 7.0, 6.0                 | 7.0, 6.0, 3.1                     |
```

Comment:
3.1 is out of support now. Remove from the table?

Reply:
It's available in the feed. It's not listed in the "Supported" column.

Reply:
What's our policy for maintaining documentation for versions that are no longer supported? Should we delete it instead? FWIW, it is what I have been doing in my doc edits.

Also, this makes it sound like that these no longer supported feeds are going to be available forever. I doubt that it is the case. I expect that we are going to turn them off at some point.

Reply:
I think when/if they get turned off, we can remove it from the docs. We've been rewriting docs for the latest versions of .NET too, but I don't think we shouldn't purge old versions from the installation docs specifically. While we may not focus on it via the examples and install instructions, it's reasonable to still call it out as available via X Y or Z methods.

Reply:
I still think that mentions of unsupported versions add unnecessary clutter to the docs. It is fine to make information about installation of no longer supported .NET versions hard to discover.

Reply:
I'll politely disagree as I feel it has merit being there. However, if it causes issues with users or confusion, I'll remove it, but I think it's useful information.

Reply:
I think it is safe to say that this is not useful information for 99.99% of the target audience of this documentation.

My overall feel about this documentation is that it is too complicated, especially if you compare it with what the equivalent documentation looks like for other platforms (for example, compare our documentation with the simplicity of https://go.dev/doc/install). I have been thinking about how to simplify it and what to remove. The information about availability of out-of-support versions sounded like a no-brainer to remove to me.